static/css/style.css

```diff
@@ -33,6 +33,15 @@ a:hover {
     text-decoration: none;
 }
 
+/* Global content border/shadow */
+.neu-shadow {
```

Comment:
what does `neu` stand for?

Reply:
It basically stands for my lack of experience 😆  have changed it to .main-content-shadow